content/millennium/r4/individuals/practitioner.md

```diff
@@ -37,9 +37,14 @@ Search for Practitioners that meet supplied query parameters:
 
 ### Parameters
 
- Name  | Required? | Type      | Description
--------|-----------|-----------|-------------------------------------------------------
- `_id` | Yes       | [`token`] | The logical resource id associated with the resource.
+ Name        | Required?          | Type      | Description
+-------------|--------------------|-----------|-------------------------------------------------------
+ `_id`       | This or identifier | [`token`] | The logical resource id associated with the resource.
+ `identifier`| This or _id        | [`token`] | A practitioner identifier/alias.Example identifier=<http://hl7.org/fhir/sid/us-npi>&#124;4326587548
```

Comment:
Need a space between the sentences in the description

Reply:
Also, based on what I'm seeing in other resources, we don't provide the identifier= portion in the example.

Should just be `http://hl7.org/fhir/sid/us-npi|4326587548`

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/commit/9269a0c450734ce07b1305892f391e83173714a6

Reply:
The example identifier value( http://hl7.org/fhir/sid/us-npi|4326587548) still needs to be monospaced.

You can do that using backticks \` \`

Reply:
https://github.com/cerner/fhir.cerner.com/commit/a18d433eaff97eab569c37412a3e7a232baec926